azure-pipelines.yml

```diff
@@ -12,43 +10,8 @@ strategy:
   matrix:
     Python310-onnx:
       python.version: '3.10'
-      onnx.standard: '1'
-      test.examples: '1'
 steps:
 - task: UsePythonVersion@0
   inputs:
     versionSpec: '$(python.version)'
   displayName: 'Use Python $(python.version)'
```

Comment:
I might miss some context here: is it intended that after this PR this azure-pipelines seems to test nothing?

Reply:
Yes. We just needed it so ms can inject mandatory checks